### PR TITLE
Create ui-c8y-1020-18-3-translation-added-from-public-options-will-now-have-higher-priority-than.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-60268](https://cumulocity.atlassian.net/browse/MTM-60268)
Original PR: [6932](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6932)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-18-3-translation-added-from-public-options-will-now-have-higher-priority-than.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Translation added from public options will now have higher priority than plugin po files (#6932)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-18-3-translation-added-from-public-options-will-now-have-higher-priority-than.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Public option translations now have higher priority than plugin po files
```

File: content/change-logs/application-enablement/ui-c8y-1020-18-3-translation-added-from-public-options-will-now-have-higher-priority-than.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60268
+version: 1020.18.3
+---
+Translation added from public options will now have higher priority than plugin po files (#6932)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-18-3-translation-added-from-public-options-will-now-have-higher-priority-than.md:17]: The Claude AI suggested to use the following body:
```suggestion
Previously, translations from plugin po files took precedence over translations added via public options, which could lead to inconsistencies. With this change, translations added via public options will now have a higher priority than translations from plugin po files. This ensures that custom translations specified in the public options are always used, providing a more consistent and predictable translation behavior for users. Existing translations in plugin po files that conflict with public option translations will be overridden.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by reey — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-18-3-translation-added-from-public-options-will-now-have-higher-priority-than.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60268
+version: 1020.18.3
+---
+Previously, translations from plugin po files took precedence over translations added via public options, which could lead to inconsistencies. With this change, translations added via public options will now have a higher priority than translations from plugin po files. This ensures that custom translations specified in the public options are always used, providing a more consistent and predictable translation behavior for users. Existing translations in plugin po files that conflict with public option translations will be overridden.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-18-3-translation-added-from-public-options-will-now-have-higher-priority-than.md:17]:
```suggestion
Previously, translations from plugin PO files (file format used for UI translations) took precedence over translations added via public options, which could lead to inconsistencies. With this change, translations added via public options will now have a higher priority than translations from plugin PO files. Existing translations in plugin PO files that conflict with public option translations will be overridden. This ensures a consistent and predictable translation behavior. 
```

File: content/change-logs/application-enablement/ui-c8y-1020-18-3-translation-added-from-public-options-will-now-have-higher-priority-than.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Public option translations now have higher priority than plugin po files
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-18-3-translation-added-from-public-options-will-now-have-higher-priority-than.md:3]:
```suggestion
title: Public option translations now have higher priority than translations from plugin PO files
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@rahultsag Is this technical phrasing ok for you? Or do you have any suggestions for improvement?

**Review by rahultsag — APPROVED** [review]

**Pull request merged**